tests/integration/test_lint.py

```diff
@@ -0,0 +1,96 @@
+"""Check scaffolded content integration with ansible-lint."""
+
+import re
+import subprocess
+import sys
+
+from pathlib import Path
+
+import pytest
+
```

Comment:
```suggestion


@pytest.fixture(autouse=True)
def _demand_non_ansi_cli_output(monkeypatch: pytest.MonkeyPatch) -> None:
    monkeypatch.delenv("FORCE_COLOR", raising=False)
    monkeypatch.setenv("NO_COLOR", "1")
```

This is where `FORCE_COLOR` is coming from, messing with your tests: https://github.com/ansible/team-devtools/blob/de9c14c/.github/workflows/tox.yml. No need to post-process output if it's never broken in the first place.